<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Diaszano</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="EntrarCadastrar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(42, 42, 42);
border-radius: 15px;
border-style: outset;
border-width: 2px;
border-color: black;
font: 25 italic 12pt &quot;Ubuntu&quot;;

</string>
    </property>
    <property name="text">
     <string>Entrar nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomeCadastrar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Digite o seu nome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VoltarCadastrar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(42, 42, 42);
border-radius: 15px;
border-style: outset;
border-width: 2px;
border-color: black;
font: 25 italic 12pt &quot;Ubuntu&quot;;

</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
